server/bin/pbench-sync-satellite.sh

```diff
@@ -250,10 +253,26 @@ for host in $hosts ;do
         for x in $state_list; do
             echo "$host/TO-SYNC/$x" >> ${state_change_log}
         done
+        rm $logdir/$host/ok-checks.log
+    fi
+
+    if [[ ! -s $logdir/$host/fail-checks.log ]] ; then
+        rm $logdir/$host/fail-checks.log
+    fi
+
+    if [[ -z $(grep -q -E -v 'OK|FAILED' $logdir/$host/md5-checks.log) ]] ; then
+        rm $logdir/$host/md5-checks.log
     fi
+
     popd > /dev/null 2>&4
 done
 
+rmdir ${logdir}/${host} 2>/dev/null
+sts = ${?}
+if [[ ${sts}!=0 ]]; then
+    echo "$PROG: rmdir failed to remove ${logdir}/${host} - status ${sts}" s/&>4/>&4/
```

Comment:
The characters at the end of the line are not valid `bash` syntax: `s/&>4/>&4/`.

The previous comment where that was written was intended to be short hand for substituting "`&>4`" with "`>&4`", the latter being the correct `bash` shell syntax for writing the text of the `echo` command to file descriptor `#4`.